website/content/docs/k8s/annotations-and-labels.mdx

```diff
@@ -0,0 +1,171 @@
+
+    ```yaml
+    annotations:
+      "consul.hashicorp.com/connect-service-upstreams":"[service-name]:[port]:[optional datacenter]"
+    ```
+
+  - Consul Enterprise Namespaces
+
+    If running Consul Enterprise 1.7+, your upstream services may be running in different
+    namespaces. The upstream namespace can be specified after the service name
+    as `[service-name].[namespace]`. See [Consul Enterprise Namespaces](#consul-enterprise-namespaces)
+    below for more details on configuring the injector.
+
+    ```yaml
+    annotations:
+      "consul.hashicorp.com/connect-service-upstreams":"[service-name].[service-namespace]:[port]:[optional datacenter]"
+    ```
+
+    -> **NOTE:** If the namespace is not specified it will default to the namespace
+    of the source service.
+
+    ~> **WARNING:** Setting a namespace when not using Consul Enterprise or using a version < 1.7
+    is not supported. It will be treated as part of the service name.
+
+  - [Prepared Query](/docs/connect/proxies#dynamic-upstreams-require-native-integration)
+
+    ```yaml
+    annotations:
+      'consul.hashicorp.com/connect-service-upstreams': 'prepared_query:[query name]:[port]'
+    ```
+
+  - Multiple Upstreams
+
+    If you would like to specify multiple services or upstreams, delimit them with commas
+
+    ```yaml
+    annotations:
+      "consul.hashicorp.com/connect-service-upstreams":"[service-name]:[port]:[optional datacenter],[service-name]:[port]:[optional datacenter]"
+    ```
+
+    ```yaml
+    annotations:
+      "consul.hashicorp.com/connect-service-upstreams":"[service-name]:[port]:[optional datacenter],prepared_query:[query name]:[port]"
+    ```
+
+- `consul.hashicorp.com/envoy-extra-args` - A space-separated list of [arguments](https://www.envoyproxy.io/docs/envoy/latest/operations/cli)
+  to be passed to the injected envoy binary.
+
+  ```yaml
+  annotations:
+    consul.hashicorp.com/envoy-extra-args: '--log-level debug --disable-hot-restart'
+  ```
+
+- `consul.hashicorp.com/service-tags` - A comma separated list of tags that will
+  be applied to the Consul service and its sidecar.
+
+  ```yaml
+  annotations:
+    consul.hashicorp.com/service-tags: foo,bar,baz
+  ```
+
+  If you need your tag to have a comma in it you can escape the comma with `\,`. For example,
+  `consul.hashicorp.com/service-tags: foo\,bar\,baz` will become the single tag `foo,bar,baz`.
+
+- `consul.hashicorp.com/service-meta-<YOUR_KEY>` - Set Consul meta key/value
+  pairs that will be applied to the Consul service and its sidecar.
+  The key will be what comes after `consul.hashicorp.com/service-meta-`, e.g.
+  `consul.hashicorp.com/service-meta-foo: bar` will result in `foo: bar`.
+
+  ```yaml
+  annotations:
+    consul.hashicorp.com/service-meta-foo: baz
+    consul.hashicorp.com/service-meta-bar: baz
+  ```
+
+- `consul.hashicorp.com/service-ignore` - This label can be set on a Kubernetes Service.
```

Comment:
Since this is the only label we have. Could we pull this out in a Label heading, and categorize annotations under the Annotations heading?